### DISCUSSION TITLE
Help needed maintaining this project

### DISCUSSION THREAD

**Discussion by mmcc-xx** [mmcc-xx/BirdCAGE#69] [Announcements]
Hi All - due to some changes at work and home, I'm unfortunately not going to be able to keep up on this project. Would anyone be interested in taking over? If so please let me know. I am not an expert in github, so I don't know if I'd somehow hand over the keys or make someone else and admin or how it would work. If you know, please let me know.

It would be great if the same person could take on WhosAtMyFeeder as well (and maybe integrate the two?)

Thanks - and my apologies for having dropped off the planet for a while.

**Comment by matthew73210** [comment]
I could have a go, no point letting this thing rot

**Reply by majagu** [reply]
Hey @matthew73210  - looks like the link to your fork 404's now.  I take it you gave up?
